Makefile

```diff
@@ -450,6 +450,29 @@ build-image-debug: ## Build a docker image as specified in the Dockerfile.debug
 		--build-arg SCCACHE_REGION=$(SCCACHE_REGION) \
 		--build-arg SCCACHE_ENDPOINT=$(SCCACHE_ENDPOINT)
 
+.PHONY:build-stress-test-image
+build-stress-test-image: ## Build a docker image as specified in the Dockerfile
+	$(DOCKER) build ./utils/stress_test -t $(REPO)-stress-test:$(IMAGE_TAG) \
+		$(PULL_OPTS) \
+		--progress=plain \
+		--platform=linux/${DEB_ARCH_NAME_${ARCH}} \
+		--secret id=aws,src=$(AWS_SHARED_CREDENTIALS_FILE) \
+		--rm \
+		--build-arg BUILD_ENVS="$(BUILD_ENVS)" \
+		--build-arg BUILD_IMAGE=$(RUST_IMAGE) \
+		--build-arg TARGET=$(TARGET) \
+		--build-arg TARGET_DIR=$(TARGET_DIR) \
+		--build-arg TARGET_ARCH=$(ARCH) \
+		--build-arg BUILD_TIMESTAMP=$(BUILD_TIMESTAMP) \
+		--build-arg BUILD_VERSION=$(BUILD_VERSION) \
+		--build-arg FEATURES='$(FEATURES_ARG)' \
+		--build-arg REPO=$(REPO) \
+		--build-arg VCS_REF=$(VCS_REF) \
+		--build-arg VCS_URL=$(VCS_URL) \
+		--build-arg SCCACHE_BUCKET=$(SCCACHE_BUCKET) \
+		--build-arg SCCACHE_REGION=$(SCCACHE_REGION) \
+		--build-arg SCCACHE_ENDPOINT=$(SCCACHE_ENDPOINT)
```

Comment:
I don't think any of these build args are utilized by the stress test Dockerfile and can be removed